src/funding/paypal/template.jsx

```diff
@@ -158,6 +165,7 @@ export function Label(opts : LabelOptions) : ChildType {
 
 export function WalletLabelOld(opts : WalletLabelOptions) : ?ChildType {
     const { logoColor, instrument, locale, content, commit } = opts;
+    const nonce = opts.nonce || '';
```

Comment:
Let's have a single place where we default the nonce to `''`, and make it a mandatory string everywhere else. Want to avoid many places having to default the value to `''`

Reply:
@bluepnume What do you think about defaulting nonce to an empty string here in sdk-client? https://github.com/paypal/paypal-sdk-client/blob/e2074894e776aa55881833c07158d1f66f70dc0d/src/script.js#L242. Currently it returns `undefined` if no nonce is set with data-csp-nonce attribute. Wondering if that should just default to empty string.

Reply:
Yup, that sounds good to me!